dnsapi/dns_dp.sh

```diff
@@ -199,7 +199,7 @@ _get_root() {
 
 #Usage: method  URI  data
 _rest() {
-  m=$1
+  export m=$1		# e.g. GET|POST
```

Comment:
this is not true: check it again: $m is just not used in this function, but in others...